src/solver/optimisation/opt_construction_matrice_des_contraintes_cas_lineaire.cpp

```diff
@@ -229,16 +237,15 @@ void OPT_ConstruireLaMatriceDesContraintesDuProblemeLineaire(PROBLEME_HEBDO* pro
                     Colonne[nombreDeTermes] = var;
                     nombreDeTermes++;
 
-                    if (exportStructure)
-                    {
-                        OPT_Export_add_variable(
-                          varname,
-                          var,
-                          Enum::ExportStructDict::ValeurDeNTCOrigineVersExtremite,
-                          timeStepInYear, // TODO[FOM] remove
-                          pays,
-                          interco);
-                    }
+                    const auto origin
+                      = problemeHebdo
+                          ->NomsDesPays[problemeHebdo->PaysOrigineDeLInterconnexion[interco]];
+                    const auto destination
+                      = problemeHebdo
+                          ->NomsDesPays[problemeHebdo->PaysExtremiteDeLInterconnexion[interco]];
+                    currentAssetsStorage.origin = origin;
+                    currentAssetsStorage.destination = destination;
+                    variableNamer.NTCValueOriginToDestination(var);
```

Comment:
About : `variableNamer.NTCValueOriginToDestination(var)`
Method should be renamed into **NTCdirect** and maybe used as follows : 
```C++
variableNamer.NTCdirect(var, origin, destination);
```